main.rb

```diff
@@ -0,0 +1,168 @@
+require_relative './person'
+require_relative './student'
+require_relative './rental'
+require_relative './teacher'
+require_relative './book'
+require_relative './classroom'
+
+class App
+  def initialize
+    @books = []
+    @rentals = []
+    @people = []
+  end
+
+  def actions
+    puts
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
+  end
+
+  def start
+    puts 'Welcome to School Library App!'
+
+    loop do
+      actions
+
+      option = gets.chomp
+
+      break if option == '7'
+
+      handle_action(option)
+    end
+
+    puts 'Thank you for using this app!'
+  end
+
+  private
+
+  def handle_action(option)
+    case option
+    when '1'
+      list_books
+    when '2'
+      list_people
+    when '3'
+      create_person
+    when '4'
+      create_book
+    when '5'
+      create_rental
+    when '6'
+      list_rentals
+    else
+      puts 'That is not a valid option'
+    end
+  end
+
+  def list_books
+    @books.each { |book| puts book }
+  end
+
+  def list_people
+    @people.each { |person| puts person }
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    choice = gets.chomp
+
+    case choice
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    else
+      puts 'That is not a valid input'
+      nil
+    end
+  end
+
+  def create_student
+    print 'Age: '
+    age = gets.chomp
+
+    print 'Name: '
+    name = gets.chomp
+
+    print 'Has parent permission? [Y/N]: '
+    parent_permission = gets.chomp.downcase == 'y'
+
+    student = Student.new(classroom: @classroom, age: age, name: name, parent_permission: parent_permission)
+    @people.push(student)
+
```

Comment:
- [ ] Since the student is already created you don't have to initialize a new variable student, and also the params inside, and the downcase is not accepted also. please fix this.